src/components/Form/formFunctions.ts

```diff
@@ -95,7 +95,14 @@ export const onChangeText = <SpecificFields extends Fields>(
 ): SelectChangeEventHandler => {
     return (event) => {
         const input = event.target.value;
-        const errorType = getErrorType(input, required, regex, additionalCondition);
+        const errorType = getErrorType(
+            key === "telephoneNumber" || key === "phoneNumber"
```

Comment:
Just as a note, we could also do something like `key.includes("phone")` here. I think we want to keep it explicit, but there might be situations where the more general solution is appropriate.